generator/src/main/java/com/graphicsfuzz/generator/fuzzer/OpaqueExpressionGenerator.java

```diff
@@ -246,11 +238,41 @@ public OpaqueExpressionGenerator(IRandom generator, GenerationParams generationP
                           makeOpaqueOne(type, constContext, depth, fuzzer),
```

Comment:
There's no need to clone shiftValue here, as it's not used multiple times.  (You needed to clone it before as you were using it twice).